hbase-client/src/main/java/org/apache/hadoop/hbase/client/Get.java

```diff
@@ -260,30 +228,6 @@ public Get setTimestamp(long timestamp) {
     return (Get) super.setColumnFamilyTimeRange(cf, minStamp, maxStamp);
   }
 
-  /**
-   * Get all available versions.
-   * @return this for invocation chaining
-   * @deprecated It is easy to misunderstand with column family's max versions, so use
-   *             {@link #readAllVersions()} instead.
-   */
-  @Deprecated
-  public Get setMaxVersions() {
-    return readAllVersions();
-  }
-
-  /**
-   * Get up to the specified number of versions of each column.
-   * @param maxVersions maximum versions for each column
-   * @throws IOException if invalid number of versions
-   * @return this for invocation chaining
-   * @deprecated It is easy to misunderstand with column family's max versions, so use
```

Comment:
For sure deprecated since 2.0.0?

Reply:
Deprecated with HBASE-17125, which has a fix version set to 2.0.0.